src/clr-core/button/button.element.ts

```diff
@@ -34,162 +32,58 @@ export enum ClrLoadingState {
- * @cssprop --clr-btn-danger-hover-bg-color
- * @cssprop --clr-btn-danger-hover-color
- * @cssprop --clr-btn-danger-box-shadow-color
- * @cssprop --clr-btn-danger-disabled-color
- * @cssprop --clr-btn-danger-disabled-bg-color
- * @cssprop --clr-btn-danger-disabled-border-color
- * @cssprop --clr-btn-danger-checked-bg-color
- * @cssprop --clr-btn-danger-checked-color
- * @cssprop --clr-btn-danger-outline-color
- * @cssprop --clr-btn-danger-outline-bg-color
- * @cssprop --clr-btn-danger-outline-border-color
- * @cssprop --clr-btn-danger-outline-hover-bg-color
- * @cssprop --clr-btn-danger-outline-hover-color
- * @cssprop --clr-btn-danger-outline-box-shadow-color
- * @cssprop --clr-btn-danger-outline-disabled-color
- * @cssprop --clr-btn-danger-outline-disabled-bg-color
- * @cssprop --clr-btn-danger-outline-disabled-border-color
- * @cssprop --clr-btn-danger-outline-checked-bg-color
- * @cssprop --clr-btn-danger-outline-checked-color
- * @cssprop --clr-btn-link-color
- * @cssprop --clr-btn-link-bg-color
- * @cssprop --clr-btn-link-border-color
- * @cssprop --clr-btn-link-hover-bg-color
- * @cssprop --clr-btn-link-hover-color
- * @cssprop --clr-btn-link-disabled-color
- * @cssprop --clr-btn-link-disabled-bg-color
- * @cssprop --clr-btn-link-disabled-border-color
- * @cssprop --clr-btn-link-checked-bg-color
- * @cssprop --clr-btn-link-checked-color
- * @cssprop --clr-btn-inverse-color
- * @cssprop --clr-btn-inverse-border-color
- * @cssprop --clr-btn-inverse-bg-color
- * @cssprop --clr-btn-inverse-hover-bg-color
- * @cssprop --clr-btn-inverse-hover-color
- * @cssprop --clr-btn-inverse-box-shadow-color
- * @cssprop --clr-btn-inverse-disabled-color
- * @cssprop --clr-btn-inverse-disabled-bg-color
- * @cssprop --clr-btn-inverse-disabled-border-color
- * @cssprop --clr-btn-inverse-checked-bg-color
- * @cssprop --clr-btn-inverse-checked-color
- * @cssprop --clr-btn-icon-disabled-color
- * @cssprop --clr-btn-group-focus-outline
+ * @cssprop --box-shadow-color
+ * @cssprop --border-radius
+ * @cssprop --border-width
+ * @cssprop --font-size
+ * @cssprop --font-weight
+ * @cssprop --border-color
+ * @cssprop --color
+ * @cssprop --background
+ * @cssprop --line-height
+ * @cssprop --letter-spacing
  */
 // @dynamic
 export class CwcButton extends CwcBaseButton {
-  /** Define the type of action the button triggers */
+  /**
+   * Define the type of action the button triggers
+   *
+   * - `solid`: buttons direct the user’s attention to the primary action the application is suggesting that the user take.
+   * - `outline`: buttons indicate secondary actions that compliments a primary action or reduces visual noise when there are many actions on the page.
+   * - `flat`: buttons are used as tertiary buttons. Can also be used inline because they are different from content in style and recognizable as buttons alongside content.
+   */
   @property({ type: String })
-  action: 'default' | 'outline' | 'link';
+  action: 'solid' | 'outline' | 'flat';
```

Comment:
Updated to match how the docs describe the buttons as well as what was discussed in slack.